lib/hex_web/web/templates/docs/mirrors.html.eex

```diff
@@ -15,7 +15,13 @@
 </p>
 
 <p>
-  <strong>Example</strong>: <code>$ mix hex.config cdn_url https://s3.amazonaws.com/s3.hex.pm</code>
+  <strong>Mix Example</strong>: <code>$ mix hex.config cdn_url https://s3.amazonaws.com/s3.hex.pm</code>
+</p>
+<p>
+  <strong>Rebar3 Example</strong>: <code>{rebar_packages_cdn, "https://s3.amazonaws.com/s3.hex.pm"}.</code>
+</p>
```

Comment:
I don't think it is clear where this should be configured. Is there any reason to have it over ` rebar3 hex config`?